### ISSUE TITLE
Show samples of each menu on the home page

### ISSUE THREAD

**Issue by VashJuan** [MarkXA/ddrmenutemplates#5]
Many people won't take time to install and explore the options of these many great menu systems. Who ever methodically goes through the install of each of these should take some screenshots, so that 1st timers (like me) have an idea of the possible menu styles available from each of these. 

Also some folks may not have a clue about what this project is for: a basic here's where to get DNN info, and that this is a repository for old and new menu styles for DNN, might be a nice addition to the home page's readme.md.

**Comment by Paul-Morris** [comment]
Agreed. I can't for the life of me get this to work! I add a ddr menu to my page using dnn 9 **Add module > DDR Module**.  All I did was type the path "ddrmenutemplates/Accordion" to the template in the "menu style" field. It works fine with the default menu but I can't seem to add anything new.

Anyone have some tips? I get this error:

`Error: DDR Menu is currently unavailable. DotNetNuke.Services.Exceptions.ModuleLoadException: Couldn't load menu style 'ddrmenutemplates/Accordion': System.ArgumentException: Illegal characters in path. at System.Security.Permissions.FileIOPermission.CheckIllegalCharacters(String[] str, Boolean onlyCheckExtras) at System.Security.Permissions.FileIOPermission.AddPathList(FileIOPermissionAccess access, AccessControlActions control, String[] pathListOrig, Boolean checkForDuplicates, Boolean needFullPath, Boolean copyPathList) at System.Security.Permissions.FileIOPermission..ctor(FileIOPermissionAccess access, String path) at System.Web.InternalSecurityPermissions.PathDiscovery(String path) at System.Web.HttpRequest.MapPath(VirtualPath virtualPath, VirtualPath baseVirtualDir, Boolean allowCrossAppMapping) at System.Web.HttpServerUtility.MapPath(String path) at DotNetNuke.Web.DDRMenu.TemplateEngine.TemplateDefinition.FromManifest(String manifestUrl) at DotNetNuke.Web.DDRMenu.TemplateEngine.TemplateDefinition.FromName(String templateName, String manifestName) at DotNetNuke.Web.DDRMenu.MenuBase.Instantiate(String menuStyle) ---> System.ApplicationException: Couldn't load menu style 'ddrmenutemplates/Accordion': System.ArgumentException: Illegal characters in path. at System.Security.Permissions.FileIOPermission.CheckIllegalCharacters(String[] str, Boolean onlyCheckExtras) at System.Security.Permissions.FileIOPermission.AddPathList(FileIOPermissionAccess access, AccessControlActions control, String[] pathListOrig, Boolean checkForDuplicates, Boolean needFullPath, Boolean copyPathList) at System.Security.Permissions.FileIOPermission..ctor(FileIOPermissionAccess access, String path) at System.Web.InternalSecurityPermissions.PathDiscovery(String path) at System.Web.HttpRequest.MapPath(VirtualPath virtualPath, VirtualPath baseVirtualDir, Boolean allowCrossAppMapping) at System.Web.HttpServerUtility.MapPath(String path) at DotNetNuke.Web.DDRMenu.TemplateEngine.TemplateDefinition.FromManifest(String manifestUrl) at DotNetNuke.Web.DDRMenu.TemplateEngine.TemplateDefinition.FromName(String templateName, String manifestName) at DotNetNuke.Web.DDRMenu.MenuBase.Instantiate(String menuStyle) at DotNetNuke.Web.DDRMenu.MenuBase.Instantiate(String menuStyle) at DotNetNuke.Web.DDRMenu.MenuView.OnPreRender(EventArgs e) --- End of inner exception stack trace ---`
